src/main/resources/CU-ApplicationResources.properties

```diff
@@ -528,6 +528,7 @@ error.concur.token.retet.production=Resetting the access token to an emty string
 error.cbcp.batch.upload.email.subject=Error occurred during Corporate Billed Corporate Paid batch upload process.
 error.cbcp.batch.upload.email.body.starter=Errors occured during the Corporate Billed Corporate Paid upload process.
 error.create.accounting.document.attachment.download=Could not download the attachment from {0}
+<<<<<<< HEAD
```

Comment:
Looks like there is something left over from there merge. Is this okay?

Reply:
yep, I will fix .... don't merge this PR yet, I haven't done any validation yet